set.mm

```diff
@@ -78074,21 +78072,21 @@ currently used conventions for such cases (see ~ cbvmpox , ~ ovmpox and
     wfrlem14 $p |- ( z e. ( A \ dom F ) ->
                      ( y e. ( dom F u. { z } ) ->
                        ( C ` y ) = ( G ` ( C |` Pred ( R , A , y ) ) ) ) ) $=
-      ( cv wcel csn cun cfv cres wceq ax-mp c0 cdm wfn cpred wi wfrlem13 weq wo
-      cdif elun velsn orbi2i bitri wa wfrlem12 wfun wb fnfun wss ssun1 sseqtrri
-      cop funssfv wfrdmcl fun2ssres syl3an3 fveq2d eqeq12d mp3an2 sylan syl5ibr
-      w3a ex pm2.43d vsnid elun2 reseq1i resundir wfr wn wwe predfrirr ressnop0
-      wefr mp2b uneq2i un0 eqtri 3eqtri fveq2i opex elsn mpbir eleqtrri fnopfvb
-      opeq2i mpbiri mpan2 fveq2 predeq3 reseq2d syl5ibrcom jaod syl5bi syl ) BL
-      ZCFUAZUHMDXFXENZOZUBZALZXHMZXJDPZDCEXJUCZQZGPZRZUDBCDEFGHIJKUEXKXJXFMZABU
-      FZUGZXIXPXKXQXJXGMZUGXSXJXFXGUIXTXRXQAXEUJUKULXIXQXPXRXIXQXPXIXQXQXPUDXQX
-      PXIXQUMXJFPZFXMQZGPZRZACEFGHIJUNXIDUOZXQXPYDUPZXHDUQYEFDURZXQYFFFXEFCEXEU
-      CZQZGPZVAZNZOZDFYLUSKUTYEYGXQVKZXLYAXOYCXJDFVBYNXNYBGXQYEYGXMXFURXNYBRCEF
-      GXJJVCXMDFVDVEVFVGVHVIVJVLVMXIXPXRXEDPZDYHQZGPZRZXIXEXHMZYRXEXGMYSBVNXEXG
-      XFVOSXIYSUMYRXEYQVAZDMYTYMDYTYLMZYTYMMUUAYTYKRYQYJXEYPYIGYPYMYHQYIYLYHQZO
-      ZYIDYMYHKVPFYLYHVQUUCYITOYIUUBTYICEVRZXEYHMVSUUBTRCEVTUUDHCEWCSCEXEWAXEYJ
-      YHWBWDWEYIWFWGWHWIWOYTYKXEYQWJWKWLYTYLFVOSKWMXHXEYQDWNWPWQXRXLYOXOYQXJXED
-      WRXRXNYPGXRXMYHDCEXJXEWSWTVFVGXAXBXCXD $.
+      ( cv wcel csn cun cfv cres wceq ax-mp c0 cdm cdif wfn cpred wfrlem13 elun
+      wi wo velsn orbi2i bitri wa wfrlem12 wfun wb fnfun wss cop ssun1 sseqtrri
+      w3a funssfv wfrdmcl fun2ssres syl3an3 fveq2d eqeq12d mp3an2 sylan syl5ibr
+      pm2.43d vsnid elun2 reseq1i resundir wfr wwe wefr predfrirr ressnop0 mp2b
+      ex uneq2i un0 eqtri 3eqtri fveq2i opeq2i opex elsn mpbir eleqtrri fnopfvb
+      wn mpbiri mpan2 fveq2 predeq3 reseq2d syl5ibrcom jaod syl5bi syl ) BLZCFU
+      AZUBMDXEXDNZOZUCZALZXGMZXIDPZDCEXIUDZQZGPZRZUGBCDEFGHIJKUEXJXIXEMZXIXDRZU
+      HZXHXOXJXPXIXFMZUHXRXIXEXFUFXSXQXPAXDUIUJUKXHXPXOXQXHXPXOXHXPXPXOUGXPXOXH
+      XPULXIFPZFXLQZGPZRZACEFGHIJUMXHDUNZXPXOYCUOZXGDUPYDFDUQZXPYEFFXDFCEXDUDZQ
+      ZGPZURZNZOZDFYKUSKUTYDYFXPVAZXKXTXNYBXIDFVBYMXMYAGXPYDYFXLXEUQXMYARCEFGXI
+      JVCXLDFVDVEVFVGVHVIVJWBVKXHXOXQXDDPZDYGQZGPZRZXHXDXGMZYQXDXFMYRBVLXDXFXEV
+      MSXHYRULYQXDYPURZDMYSYLDYSYKMZYSYLMYTYSYJRYPYIXDYOYHGYOYLYGQYHYKYGQZOZYHD
+      YLYGKVNFYKYGVOUUBYHTOYHUUATYHCEVPZXDYGMWNUUATRCEVQUUCHCEVRSCEXDVSXDYIYGVT
+      WAWCYHWDWEWFWGWHYSYJXDYPWIWJWKYSYKFVMSKWLXGXDYPDWMWOWPXQXKYNXNYPXIXDDWQXQ
+      XMYOGXQXLYGDCEXIXDWRWSVFVGWTXAXBXC $.
```

Comment:
This shortening is insignificant.  For some reason a weq is spared from the list of used theorems in the new proof,  even though = is used between setvars. Otherwise I see no change.  I checked the proof lines optically line by line, and saw no difference whatsoever.  No change in proof byte count.  MM accepts and verifies both proofs alike.

Reply:
> For some reason a weq is spared from the list of used theorems in the new proof, even though = is used between setvars.

Can you provide a step number which uses equality between setvars? I looked a bit into the proof and so far I only found equality between classes (therefore making `wceq` sufficient and `weq` not needed).

Reply:
Step 7 velsn: (𝑦 ∈ {𝑧} ↔ 𝑦 = 𝑧)

Reply:
Indeed, `class y` and `class z` were already proved in earlier steps, so that is reused, and only wceq is needed, not weq.

Reply:
It is in my eyes a matter of taste, whether you always interpret a formula 𝑦 = 𝑧 as an equality between sets (weq), and in case you need this equality as one between classes (wceq), or if it is delivered as such, you need a conversion step.  See the comment of weq: However, logically weq is considered to be a primitive syntax. Otherwise you silently apply the overload as a kind of hidden knowlwdge to switch between the possible primitives back and forth without control, or lets better say: proof step.  This has no practical implications, as the result is always the same.  In the end it is a matter of one's liking, I'd say.

Reply:
My above comment describes the modified proof. It is only a description.